### PR TITLE
MQTT path to ENV

### PR DESCRIPTION
**Pull request by disruptivepatternmaterial**
really basic no checking for default or extra /

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ const initialize = async () => {
   client.on("connect", () => {
     console.info("Connected to MQTT broker");
 
-    client.publish("unifi/protect-downloader/availability", "online", {
+    client.publish("${MQTT_PATH}/availability", "online", {
```
Comment (bostaunieux) [inline src/index.ts:101]: Need to switch double quotes to backticks for string interpolation

File: src/index.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import EventProcessor from "./event_processor";
 import { CameraDetails, CameraId, isMotionEndEvent } from "./types";
 import VideoDownloader from "./video_downloader";
 
-const { CAMERAS, DOWNLOAD_PATH, PREFER_SMART_MOTION, MQTT_HOST, UNIFI_HOST, UNIFI_USER, UNIFI_PASS } = process.env;
+const { CAMERAS, DOWNLOAD_PATH, PREFER_SMART_MOTION, MQTT_HOST, UNIFI_HOST, UNIFI_USER, UNIFI_PASS, MQTT_PATH } = process.env;
```
Comment (bostaunieux) [inline src/index.ts:9]: Good idea, but should have a default value. Something like
```
const MQTT_PATH = process.env.MQTT_PATH ?? "unifi/protect-downloader/";
```

File: src/index.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ if (!UNIFI_HOST || !UNIFI_USER || !UNIFI_PASS) {
 const initialize = async () => {
   const client: Client = mqtt.connect(MQTT_HOST, {
     will: {
-      topic: "unifi/protect-downloader/availability",
+      topic: "${MQTT_PATH}/availability",
```
Comment (bostaunieux) [inline src/index.ts:22]: Same comment about using backticks

File: src/index.ts (hunk 2 above)
Comment (disruptivepatternmaterial) [reply src/index.ts:9]: agree, I will work on it some more this week

**Comment by bostaunieux** [comment]
Closing due to inactivity.

**Pull request closed without merging**